### PR TITLE
fix: Add geo/ISP lookup button config option to README

### PR DESCRIPTION
**Pull request by joshtrichards**
This was implemented in #535.

### CODE REVIEW

**Review by st3iny — APPROVED** [review]

**Comment by st3iny** [comment]
CI failures are unrelated.

File: README.md [hunk 1]
```diff
@@ -93,6 +93,11 @@ via an app password as suspicious. Should it detect a password login where the (
 uid) is classified as suspicious by the trained model, it will add an entry to the
 ``suspicious_login`` table, including the timestamp, request id and URL.
 
+## Configuration
+
+Enable a button in notifications to lookup more info about a suspicious IP (geography, ISP):
+
+`occ config:app:set --value 1 suspicious_login show_more_info_button`
```
Comment (szaimen) [inline README.md:100]: This is the default iirc so rather document how to disable the feature?

Comment (st3iny) [reply README.md:100]: Right, it is enabled by default. @joshtrichards Could you please invert the README in another PR?

https://github.com/nextcloud/suspicious_login/blob/30b748f6f3469026914888fa552f9f44004de7df/lib/Notifications/Notifier.php#L89

Comment (joshtrichards) [reply README.md:100]: Good point. #876

**Pull request merged**